modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mmaker-source-connector.adoc

```diff
@@ -73,6 +89,81 @@ For example, if the source event has a timestamp `2023-05-22 17:00`, then this
 ====
 
+=== Advanced MirrorMaker2 Source connector configuration
+
+In most instances, the preceding basic configuration properties are sufficient.
+If you require additional property settings, then specify any of the following
+_optional_ advanced connector configuration properties by selecting *Show advanced options*
+on the *Create Connector* page:
+
+|===
+| Property name | Property key | Description
+
+| `Source cluster SSL custom certificate`
+| `source.cluster.ssl.truststore.certificates`
+| Trusted certificates in the PEM format.
+
+| `Source cluster SSL keystore key`
+| `source.cluster.ssl.keystore.key`
+| Private key in the PEM format.
+
+| `Source cluster SSL keystore certificate chain`
+| `source.cluster.ssl.keystore.certificate.chain`
+| Certificate chain in the PEM format.
+
+| `Sync topic configs interval seconds`
+| `sync.topic.configs.interval.seconds`
+| Frequency of topic config sync.
+
+| `Sync topic ACLs interval seconds`
+| `sync.topic.acls.interval.seconds`
+| Frequency of topic ACL sync.
+
+| `Topics exclude`
+| `topics.exclude`
+| Excluded topics. Supports comma-separated topic names and regexes.
+
+| `Source cluster alias`
+| `source.cluster.alias`
+| When using DefaultReplicationPolicy, topic names will be prefixed with it.
+
+| `Replication policy class`
+| `replication.policy.class`
+| Class that defines the remote topic naming convention. Use IdentityReplicationPolicy to preserve topic names. DefaultReplicationPolicy prefixes the topic with the source cluster alias.
+
+| `Replication factor`
+| `replication.factor`
+| Replication factor for newly created remote topics. Set -1 for cluster default.
+
+| `Refresh topics interval seconds`
+| `refresh.topics.interval.seconds`
+| Frequency of topic refresh.
+
+| `Offset-Syncs topic location`
+| `offset-syncs.topic.location`
+| The location (`source` or `target`) of the offset-syncs topic. The default is `source`.
+
+| `Offset-Syncs topic replication factor`
+| `offset-syncs.topic.replication.factor`
+| Replication factor for offset-syncs topic. The default is `-1`.
+
+| `Config properties exclude`
+| `config.properties.exclude`
+| Topic config properties that should not be replicated. Supports comma-separated property names and regexes.
+
+| `Compression type`
+| `producer.override.compression.type`
+| The compression type for all data generated by the producer. The default is none (no compression).
+
+| `Max size of a request`
+| `producer.override.max.request.size`
+| The maximum size of a request in bytes. The default is 1048576.
+
+| `Auto offset reset`
+| `consumer.auto.offset.reset`
+| What to do when there is no initial offset in Kafka or if the current offset does not exist any more on the server (e.g. because that data has been deleted). 'earliest' - automatically reset the offset to the earliest offset. 'latest' - automatically reset the offset to the latest offset. 'none' - throw exception to the consumer if no previous offset is found for the consumer's group.
```

Comment:
```suggestion
| What to do when there is no initial offset in Kafka, or if the current offset does not exist any more on the server (for example, because that data has been deleted). 'earliest' - automatically reset the offset to the earliest offset. 'latest' - automatically reset the offset to the latest offset. 'none' - throw exception to the consumer if no previous offset is found for the consumer's group.
```